chall03/pcuadrad.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+
+def error():
+    print ("usage: " + sys.argv[0] + " <1-9 squared_rows...>")
+    exit(1)
```

Comment:
`sys.exit()` > `exit()`.

And you could have done `sys.exit("usage: " + sys.argv[0] + " <1-9 squared_rows...>")`.

Reply:
Okay, I'll keep it in mind thanks!